src/bionics.cpp

```diff
@@ -540,6 +540,13 @@ bool player::activate_bionic( int b, bool eff_only )
             add_msg_if_player( m_info,
                                _( "You need a jumper cable connected to a vehicle to drain power from it." ) );
         }
+        if( g->u.is_wearing( "solarpack_on" ) || g->u.is_wearing ( "q_solarpack_on" ) ) {
+            add_msg_if_player( m_info, _( "Your plugged-in solar pack is now able to charge"
```

Comment:
Something strange here - why do you need line break between "charge" and "your"?

Reply:
100 characters max length of line as stated in CODE_STYLE.md ?